src/main/java/io/spine/helloworld/SimpleStorageFactory.java

```diff
@@ -0,0 +1,55 @@
+package io.spine.helloworld;
+
+import io.spine.server.ContextSpec;
+import io.spine.server.aggregate.Aggregate;
+import io.spine.server.aggregate.AggregateStorage;
+import io.spine.server.delivery.InboxStorage;
+import io.spine.server.entity.Entity;
+import io.spine.server.projection.Projection;
+import io.spine.server.projection.ProjectionStorage;
+import io.spine.server.storage.RecordStorage;
+import io.spine.server.storage.StorageFactory;
+import io.spine.server.storage.memory.InMemoryStorageFactory;
+
+/**
+ * A simple in-memory storage factory.
+ *
+ * <p>While such storage factory is sufficient for a "Hello World"-type application, it's not
+ * eligible for usage in a production environment.
+ *
+ * <p>For the production scenarios consider using
+ * <a href="https://github.com/SpineEventEngine/gcloud-java/">Spine library for
+ * Google Cloud Datastore</a>.
+ */
+final class SimpleStorageFactory implements StorageFactory {
```

Comment:
I don't like the idea of creating a storage implementation in the simplest example for Spine.

I think, we'd be better off switching the environment to "test mode". That we at least could explain to the first-time beholders.